Description: Fix a bug when opening UTF-8 files in Python3: encoding
  needs to be specified.
Author: Vincent Bernat <bernat@debian.org>
Forwarded: https://github.com/audreyr/cookiecutter/issues/27

Index: cookiecutter-0.6.1/cookiecutter/utils.py
===================================================================
--- cookiecutter-0.6.1.orig/cookiecutter/utils.py	2013-08-12 14:57:35.000000000 +0200
+++ cookiecutter-0.6.1/cookiecutter/utils.py	2013-08-17 23:44:29.835285535 +0200
@@ -37,7 +37,7 @@
     Opens a file as usual on Python 3, and with UTF-8 encoding on Python 2.
     :param filename: Name of file to open.
     """
+    kwargs['encoding'] = "utf-8"
     if PY3:
         return open(filename, *args, **kwargs)
-    kwargs['encoding'] = "utf-8"
     return codecs.open(filename, *args, **kwargs)
